plugins/com.google.cloud.tools.eclipse.util.test/src/com/google/cloud/tools/eclipse/util/ArtifactRetrieverTest.java

```diff
@@ -30,10 +29,20 @@
   private ArtifactRetriever retriever = new ArtifactRetriever();
   
   @Test
-  public void testGetGuavaLatest() {
+  public void testGetServletLatest() {
+    // group and version IDs changed with 3.0 so this should be stable
+    ArtifactVersion servlet = retriever.getLatestArtifactVersion("javax.servlet", "servlet-api");
```

Comment:
Remove the three network-using tests, since they were moved to the integration bundle.